main.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+
+from fastapi import FastAPI, Depends, HTTPException
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.security.api_key import APIKeyHeader, APIKey
+from starlette.status import HTTP_403_FORBIDDEN
+
+from nodeodm_client import NodeodmClient
+
+app = FastAPI()
+
+origins = [
+    "*"
+]
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+
+with open('config.json', 'r') as file:
+    config = json.load(file)
+
+API_KEY = config['API_KEY']
```

Comment:
Die sollten, siehe Anmerkung in der Config, aus der Umgebung kommen, sonst haben wir hier mit jeder Anpassung der Konfiguration ein neues Deployment.